### DISCUSSION TITLE
[Suggestion] Adding a spellchecker to the GitHub PR Actions

### DISCUSSION THREAD

**Discussion by tobiasfalk** [FreeCAD/FreeCAD#10687] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Making spelling errors is something completely normal, and I am also guilty of it. That is why I thought that maybe an adding a Spellchecker to the GitHub PR Action would be a nice Idea. After a quick search, I found this one [Check Spelling](https://github.com/marketplace/actions/check-spelling).

I want to know what other people think about this before doing more, maybe there is a better one.

P.s. Please count the number of spelling errors I made here.

### Full version info

```shell
.
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
We have codespell for that already: https://github.com/FreeCAD/FreeCAD/actions/workflows/codespell.yml
